scripts/SANS/sans/algorithm_detail/batch_execution.py

```diff
@@ -1151,6 +1151,8 @@ def save_workspace_to_file(workspace_name, file_formats, file_name,
         save_options.update({"TransmissionCan": transmission_can_name})
     if SaveType.NXcanSAS in file_formats:
         save_options.update({"NXcanSAS": True})
+        save_options.update({"Transmission": transmission_name})
```

Comment:
Is there a reason we add the transmission save options inside these if statements, why can we not just do it above so it would automatically apply to all save types?

Reply:
I wasn't sure if the other algorithms took them, but it would make sense to move this outside if that is the case

Reply:
I think the current implementation will force the algorithms to take them anyway if CanSAS is ticked so if they don't take them that will need to be corrected anyway.

Reply:
that's pushed now